packages/text-bitmap/src/formats/TextFormat.ts

```diff
@@ -75,39 +75,44 @@ export class TextFormat
 
         data.info.forEach((info) =>
         {
-            info.size = parseInt(info.size, 10);
+            info.size = this.parseValue(info.size);
         });
 
         data.common.forEach((common) =>
         {
-            common.lineHeight = parseInt(common.lineHeight, 10);
+            common.lineHeight = this.parseValue(common.lineHeight);
         });
 
         data.page.forEach((page) =>
         {
-            page.id = parseInt(page.id, 10);
+            page.id = this.parseValue(page.id);
         });
 
         data.char.forEach((char) =>
         {
-            char.id = parseInt(char.id, 10);
-            char.page = parseInt(char.page, 10);
-            char.x = parseInt(char.x, 10);
-            char.y = parseInt(char.y, 10);
-            char.width = parseInt(char.width, 10);
-            char.height = parseInt(char.height, 10);
-            char.xoffset = parseInt(char.xoffset, 10);
-            char.yoffset = parseInt(char.yoffset, 10);
-            char.xadvance = parseInt(char.xadvance, 10);
+            char.id = this.parseValue(char.id);
+            char.page = this.parseValue(char.page);
+            char.x = this.parseValue(char.x);
+            char.y = this.parseValue(char.y);
+            char.width = this.parseValue(char.width);
+            char.height = this.parseValue(char.height);
+            char.xoffset = this.parseValue(char.xoffset);
+            char.yoffset = this.parseValue(char.yoffset);
+            char.xadvance = this.parseValue(char.xadvance);
         });
 
         data.kerning.forEach((kerning) =>
         {
-            kerning.first = parseInt(kerning.first, 10);
-            kerning.second = parseInt(kerning.second, 10);
-            kerning.amount = parseInt(kerning.amount, 10);
+            kerning.first = this.parseValue(kerning.first);
+            kerning.second = this.parseValue(kerning.second);
+            kerning.amount = this.parseValue(kerning.amount);
         });
 
         return data;
     }
+
+    private static parseValue(value: string|number): number
+    {
+        return typeof value === 'number' ? Math.floor(value) : parseInt(value, 10);
```

Comment:
Can you explain the necessity for this new change? If this is a functional change we should add a unit-test for it.

Reply:
@bigtimebuddy It was because TS requires parseInt parameter to be of type string. Would you prefer to silence the TS error in some way instead of the change?

Reply:
What about just casting, e.g., `parseInt(kerning.amount as string)`

Reply:
Good idea, I thought that would work but tried it and get this TS error:

`Conversion of type 'number' to type 'string' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.ts(2352)`

Before I cast it as any like so: `data.kerning.forEach((kerning: any) =>`.

Reply:
If that's the case, then tslint ignore and leave the code like it was before. Would prefer to avoid `any`

Reply:
Ok, will do